lib/runner/cli/cli.js

```diff
@@ -128,9 +128,27 @@ class CliRunner {
         '  Please consult the docs at: https://nightwatchjs.org/gettingstarted#settings-file');
     }
 
+    this.validateChromeOptions();
+
     return this;
   }
 
+  validateChromeOptions() {
+    Object.keys(this.baseSettings.test_settings).map((testSettingKey) => {
+      let testSetting = this.baseSettings.test_settings[testSettingKey];
+      if (Utils.isObject(testSetting.desiredCapabilities) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions.args)) {
+        this.invalidChromeArguments = Utils.findUnquotedCliArgumentsWithWhitespaces(testSetting.desiredCapabilities.chromeOptions.args);
+
+        console.error('Unquoted argument values contains whitespace which may leads to errors : [ ' + this.invalidChromeArguments +
```

Comment:
The subject/verb agreement in this sentence is off, I think it should be
```suggestion
        console.error('Unquoted argument value contains whitespace which may lead to errors : [ ' + this.invalidChromeArguments +
```
Also, I think you want a check before this line to see if `this.invalidChromeArguments` is defined and not empty, right? If there are no invalid arguments found, you wouldn't want to output this line.

Reply:
yes you are absolute right :D